README.md

```diff
@@ -35,14 +36,22 @@ yarner create README.md
 ```
 
 This creates a file `Yarner.toml` with default settings,
-and a file `README.md` as starting point for Literate Programming.
+and a file `README.md.md` as starting point for Literate Programming
+(don't care for the double extension for now).
+
+The generated file already contains some content to get started with yarner's
+basic features. For details, read the following sections.
 
 To "compile" the project (extract code and create documentation),
 simply run:
+
 ```
 yarner
 ```
 
+This creates two sub-directories, one containing the extracted code,
```

Comment:
Maybe mention that both then need to be built using their "usual" build systems?